subt/main.py

```diff
@@ -322,11 +322,11 @@ def wait(self, dt):  # TODO refactor to some common class
     def play_system_track(self):
         print("SubT Challenge Ver1!")
         self.go_straight(2.5)  # go to the tunnel entrance
-        dist = self.follow_wall(radius=self.walldist, right_wall=self.use_right_wall, timeout=self.timeout / 2)
+        dist = self.follow_wall(radius=self.walldist, right_wall=self.use_right_wall, timeout=self.timeout)
         print("Going HOME")
         self.turn(math.radians(90), speed=-0.1)  # it is safer to turn and see the wall + slowly backup
         self.turn(math.radians(90), speed=-0.1)
-        self.follow_wall(radius=self.walldist, right_wall=not self.use_right_wall, timeout=self.timeout, dist_limit=dist + 1)
+        self.follow_wall(radius=self.walldist, right_wall=not self.use_right_wall, timeout=2.5*self.timeout , dist_limit=dist + 1)
```

Comment:
just typo - extra space in "timeout=2.5*self.timeout ,"

Reply:
thanks - fixed